modules/customizing-templates/proc-creating-a-new-software-component-using-templates.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * assemblies/assembly-configuring-templates.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="proc-creating-a-new-software-component-using-templates_{context}"]
+= Creating a new software component using Software Templates
+
+You can create a new software component using the standard templates that the platform engineers have created. The scaffolding process runs in your {product} instance.
+
+.Procedure
+
+To create a new software component, complete the following steps:
+
+. In your {product} navigation menu, click *Catalog* > *Self-service*.
+. On the *Self-service* page, click *Choose* on the *Templates* tile to initiate the scaffolding process for a template.
+
++
+[NOTE]
+====
+The information that you enter for a software template is specific to that template.
+====
+
+. Follow the wizard instructions as you enter the required details. You can choose parameters from a set of pre-defined options.
+. Optional: In the *Deployment Information* step, you have an option to *Create Workbench for OpenShift AI*.
+. In the *Review* step, verify the parameters you have entered and click *Create*.
++
+[NOTE]
+====
+* You can click *Cancel* to abort the software component creation during the template running step only if the current step supports the abort. The abort signal is then sent to a task and none of the following steps are executed. 
+* During the creation of the software component, click *Show Logs* to view the log information.
+====
+
+.Verification
+
+* If your software component is not successfully created, you can click on each section in the *Self-service* page to view the log information from the step that failed.
+
+* If your template is successfully created, you can view the component by clicking the following links on the success page:
+** *Source Repository*
+** *GitOps Repository*
+** *Open Component in Catalog*
+** *Open GitOps Resource in Catalog*
+
+* To view the software component that you have just created, click *Open Component in catalog*, or in your {product} instance, go to the *Home* page, and click *Catalog*.
+
+* To view the Source Code in your version control, click *Source Code Repository*.
```

Comment:
Consider removing fluff. These steps are repeating substeps currently at lines 39 and 41.

```suggestion
```